src/main.cpp

```diff
@@ -5773,6 +5776,35 @@ bool static ProcessMessage(CNode* pfrom, string strCommand, CDataStream& vRecv,
             return true;
         }
 
+        CNodeState *nodestate = State(pfrom->GetId());
```

Comment:
may I ask why you use a function GetId() when you could just use the variable id? i.e. pfrom->id instead of pfrom->GetId